### PR TITLE
goal, Formula

### PR DESCRIPTION
**Pull request by barkas90**
goal, Formula

### CODE REVIEW

File: Form1.cs [hunk 1]
```diff
@@ -53,5 +53,12 @@ private void textBox1_TextChanged(object sender, EventArgs e)
         {
 
         }
+        //  int Input, Expenses, Goal, Months;
```
Comment (AleksandrasMaliuginas) [inline Form1.cs:56]: Has to be uncommented : )

File: Form1.cs [hunk 2]
```diff
@@ -53,5 +53,22 @@ private void textBox1_TextChanged(object sender, EventArgs e)
...
+           // Input = int.Parse(InputBox.Text);
+           // Expenses = int.Parse(ExpensesBox.Text);
+          //  Goal = int.Parse(GoalBox.Text);
+           // Months = Goal/(Input - Expenses);
+           // MessageBox.Show(goalName + " remaining time is " + Months.ToString() + " months");
     }
+    
+            private void button2_Click(object sender, EventArgs e)
+        {
+          int Input, Expenses, Goal, Months;
```
Comment (AleksandrasMaliuginas) [inline Form1.cs:67]: It is better to name local variables in lowercase (as you named goalName)

File: Form1.cs [hunk 3]
```diff
@@ -53,5 +53,22 @@ private void textBox1_TextChanged(object sender, EventArgs e)
...
+          String goalName = goalNameBox.Text;
+          //Goal = int.Parse(goalMoney.Text);
+         // Months = Goal/(Input - Expenses);
+         // MessageBox.Show(goalName + " remaining time is " + Months.ToString() + " months");
```
Comment (AleksandrasMaliuginas) [inline Form1.cs:71]: Uncomment these too, so that on btn-click Goal info is displayed.

File: Form1.cs [hunk 4]
```diff
@@ -53,5 +53,22 @@ private void textBox1_TextChanged(object sender, EventArgs e)
...
+         this.Close();
```
Comment (AleksandrasMaliuginas) [inline Form1.cs:72]: Probably this shouldn't be here, because it closes the application. Am I right?

**Review by AleksandrasMaliuginas — CHANGES_REQUESTED** [review]

File: Form1.cs (hunk 2 above)
Comment (barkas90) [reply Form1.cs:67]: +

File: Form1.cs (hunk 3 above)
Comment (barkas90) [reply Form1.cs]: +

File: Form1.cs (hunk 4 above)
Comment (barkas90) [reply Form1.cs]: +

**Review by imila254 — APPROVED** [review]
Looks good.

**Pull request closed without merging**